### PR TITLE
Add Keadm beta e2e

### PR DESCRIPTION
**Pull request by zhu733756**
**What type of PR is this?**

Converting the former helm keadm e2e script to keadm beta e2e  script since we have supported the keadm beta commands.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by zhu733756** [comment]
Related to #3699 ,  need to rebase #3699

**Comment by zhu733756** [comment]
Need to rebase #3692

File: tests/e2e/scripts/keadm_e2e.sh [hunk 1]
```diff
@@ -87,6 +87,7 @@ END
   while true; do
       sleep 3
       kubectl get node | grep edge-node | grep -q Ready && break
+      echo
```
Comment (fisherxu) [inline tests/e2e/scripts/keadm_e2e.sh:90]: Can be removed?

**Review by fisherxu — COMMENTED** [review]
We can rebase it now :)

File: tests/e2e/scripts/keadm_e2e.sh (hunk 1 above)
Comment (zhu733756) [reply tests/e2e/scripts/keadm_e2e.sh]: Yes

**Comment by zhu733756** [comment]
> We can rebase it now :)

Done.

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @zc2638

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**